package.json

```diff
@@ -1,50 +1,46 @@
 {
-  "name": "graphql-query-rewriter",
+  "name": "@graphql-query-rewriter/core",
   "version": "0.0.1",
   "description": "",
-  "keywords": [],
-  "main": "dist/index.umd.js",
-  "module": "dist/index.es5.js",
-  "typings": "dist/types/index.d.ts",
+  "keywords": [
+    "graphql",
+    "query",
+    "rewriter",
+    "ast",
+    "schema"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist"
   ],
   "author": "David Chanin <chanindav@gmail.com>",
   "repository": {
     "type": "git",
-    "url": "https://github.com/ef-eng/graphql-query-rewriter"
+    "url": "https://github.com/ef-en/graphql-query-rewriter"
```

Comment:
this URL isn't correct - should be https://github.com/graphql-query-rewriter/core